nasa-gibs/worldview
# WV 2647 Modernize opacity slider

## Description

This modernizes the opacity slider component. Upgrading from a class component to a function component and introducing small improvements in rendering performance.

## How To Test

1. git checkout modenize-opacity-slider
2. npm ci
3. npm run watch
4. Got to `http://localhost:3000`
5. Click on "Start Comparison"
6. Click on "Opacity"
7. Test the functionality of the opacity slider at the bottom center of the screen. The layers should respond to opacity changes and the dates should update when the dates are changed in the bottom bar.
